packages/x-charts/src/models/seriesType/config.ts

```diff
@@ -60,6 +61,14 @@ export interface ChartsSeriesConfig {
     itemIdentifier: PieItemIdentifier;
     valueType: DefaultizedPieValueType;
   };
+  radar: {
+    seriesInput: DefaultizedProps<RadarSeriesType, 'id'> & { color: string };
+    series: DefaultizedRadarSeriesType;
+    seriesProp: RadarSeriesType;
+    itemIdentifier: RadarItemIdentifier;
+    valueType: number;
+    polar: true;
+  };
```

Comment:
Just curious: why do we need a `polar: true` prop here, but not for the pie chart? 

As I see it, the pie chart is also polar where the radius is constant and only the rotation axis changes, isn't it?

Reply:
I assume it is because pie chart doesn't really use polar coordinates, since it can be much simpler.